Specifications/FeelbotTalking.feature

```diff
@@ -0,0 +1,6 @@
+Scenario : FeelBotTalking1
+One scenario of FeelBot talking
+
+Given : that FeelBot is able to aknowledge words and analyze it
+When : I talk to FeelBot through my laptop or my phone
+Then : FeelBot is able to recognize the words I explain to her
```

Comment:
Given : I want to interact with the bot
When : I talk to the bot through my device
Then : I can use my voice instead of a keyboard for similar result

-------------
What there is to understand here is that once the words are expressed as written token, features handling written text tokens will work here.